app/style/Icons.less

```diff
@@ -2,6 +2,9 @@
 @icon-root: '../style/icons';
 @animation-root: '../style/animations';
 
+
+// TODO: load these as theme variables!
```

Comment:
👍

Reply:
Looks like adding this change to this PR would make extraordinarily big, will create an issue for that and I would like to work on it after merging this one!